package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -88,18 +89,23 @@ cdef extern from 'sys/types.h':
 ctypedef int fio_fd;
 ctypedef off_t fio_size_t
 
-ctypedef np.float32_t FLOAT_T
-ctypedef np.float64_t DOUBLE_T
+ctypedef float FLOAT_T
+ctypedef double DOUBLE_T
 FLOAT = np.float32
 DOUBLE = np.float64
 
 cdef enum:
     FIO_READ = 0x01
     FIO_WRITE = 0x02
 
+# exportable versions in python
 DCD_IS_CHARMM       = 0x01
 DCD_HAS_4DIMS       = 0x02
 DCD_HAS_EXTRA_BLOCK = 0x04
+# module level C decls 
+cdef  int DCD_IS_CHARMM_       = 0x01
```

Comment:
Do you have to repay the values? Can’t you only define the Constanta once?

Reply:
Constants

(typing on crappy device)

Reply:
It's to avoid cython having to juggle between the two. The C level constants can be baked in and the Python constants are used where the pythonic behaviour was required eg behaviour wrt &.

Reply:
Ok, I believe you. Was just wondering if you couldn't do something like
```python
def  int DCD_IS_CHARMM_       = 0x01

DCD_IS_CHARMM      =  DCD_IS_CHARMM_ 
```

Reply:
I will investigate.